airsonic-main/src/main/java/org/airsonic/player/spring/RegisterPrecompiledJSPInitializer.java

```diff
@@ -59,14 +59,13 @@ private static WebApp parseXmlFragment() {
                         precompiledJspWebXml),
                 IOUtils.toInputStream("</web-app>", Charset.defaultCharset()));
 
-        JAXBContext jaxbContext;
         try {
-            jaxbContext = new JAXBDataBinding(WebApp.class).getContext();
+            JAXBContext jaxbContext = new JAXBDataBinding(WebApp.class).getContext();
             Unmarshaller unmarshaller = jaxbContext.createUnmarshaller();
             WebApp webapp = (WebApp) unmarshaller.unmarshal(webXmlIS);
             try {
                 webXmlIS.close();
-            } catch (java.io.IOException e) {}
+            } catch (java.io.IOException ignored) {}
```

Comment:
very very minor: maybe could use a try with resources in the outer try? then you won't need this awkward inner try, and it'll autoclose the stream (though I don't know why the precompiledJspWebXml isn't closed here)
```
try( InputStream precompiledJspWebXml = ....; InputStream webXmlIs = ...) {
   ...
   return (WebApp) unmarshaller.unmarshal(webXmlIs); 
} catch (JAXBException e) {
    ...
} catch (IOException ignored) {}
```

Reply:
I agree, try-with-resources would be better, but my objective this pass was only to remove duplicated code.  Refactoring can be done in another PR.

Reply:
👍